service-app/internal/parser/validator.go

```diff
@@ -278,3 +283,36 @@ func (v *BaseValidator) getFieldValues(page, section, field string) (string, err
 
 	return dateStr, nil
 }
+
+func TestHelperDocumentValidation(
```

Comment:
Can this function live in `validator_test.go`? Just to keep it clean from application code.

Reply:
Since its not actually a runnable test directly IDE throws errors as a result using _test extension. I'll have to add _helper to the end.